PHP/Ex.php

```diff
@@ -0,0 +1,19 @@
+<?php
+    $a = [6, 2, 5, 10, 11, 1, 17, 20];
+
+    // Set default min and max value.
+    $min = $a[0];
+    $max = $a[0];
+
+    foreach($a as $key => $value){
+        if ($value < $min){
+            $min = $value;
+        }
+
+        if ($value > $min){
```

Comment:
actually, the output is correct, but the logic is not good at here because your compare the value with $min, it should be $value > $max